packages/did/src/Did.chain.ts

```diff
@@ -527,16 +527,16 @@ export async function generateDidAuthenticatedTx({
   blockNumber,
 }: AuthorizeCallInput & KeystoreSigningOptions): Promise<SubmittableExtrinsic> {
   const { api } = await BlockchainApiConnection.getConnectionOrConnect()
-  const signableCall =
-    new (api.registry.getOrThrow<IDidAuthorizedCallOperation>(
-      'DidDidDetailsDidAuthorizedCallOperation'
-    ))(api.registry, {
+  const signableCall = api.registry.createType<IDidAuthorizedCallOperation>(
+    api.tx.did.submitDidCall.meta.args[0].type.toString(),
```

Comment:
sneaking this in? :D
Might be already included in the merged DID refactor?!

Reply:
this is from the merge commit integrating the refactor. After Antonio's changes there and my changes here it was the last line to still use hard-coded type names for encoding (apart from testing-only code), so I decided to fix it up here for consistency.